tests/unit/testResources/flinkArtifact.ts

```diff
@@ -0,0 +1,31 @@
+import { ConnectionType } from "../../../src/clients/sidecar";
+import { CCLOUD_CONNECTION_ID } from "../../../src/constants";
+import { FlinkArtifact } from "../../../src/models/flinkArtifact";
+import { ConnectionId, EnvironmentId } from "../../../src/models/resource";
+import { TEST_CCLOUD_ENVIRONMENT_ID } from "./environments";
+
+export const TEST_CCLOUD_FLINK_ARTIFACT = createFlinkArtifact();
+
+export interface CreateFlinkArtifactArgs {
+  connectionId?: ConnectionId;
+  connectionType?: ConnectionType;
+  environmentId?: EnvironmentId;
+  id?: string;
+  name?: string;
+  description?: string;
+  provider?: string;
+  region?: string;
+}
+
+export function createFlinkArtifact(overrides: CreateFlinkArtifactArgs = {}): FlinkArtifact {
```

Comment:
We shouldn't need to duplicate the `FlinkArtifact` properties in a new interface, but if we want to keep a creation utility function, we could go with this: 
```suggestion
export function createFlinkArtifact(overrides: Partial<FlinkArtifact> = {}): FlinkArtifact {
```